### PR TITLE
Show your ENS name in the WalletButton

### PR DESCRIPTION
**Pull request by lrnt**
Show the ENS name and avatar when available. The ethers library and related packages were updated to the latest version.

image

### CODE REVIEW

File: libs/base/src/lib/context/AccountProvider.tsx [hunk 1]
```diff
@@ -126,6 +134,11 @@ const OnboardProvider: FC<{
               setAddress(account.toLowerCase())
             }
           },
+          ens: ens => {
+            if (ens) {
```
Comment (chrisjgf) [inline libs/base/src/lib/context/AccountProvider.tsx:138]:
```suggestion
            if (!!ens.name) {
```

Comment (lrnt) [reply libs/base/src/lib/context/AccountProvider.tsx]: Decided to go another way in the end.

`ens` can be undefined so it's not a good idea to check a property of `ens` without knowing whether it's undefined or not.

Comment (chrisjgf) [reply libs/base/src/lib/context/AccountProvider.tsx]: Perfect, looks good – appreciate the PR!

**Comment by chrisjgf** [comment]
LGTM 🚀 I'll leave it to Rafael to merge

**Review by rafaelugolini — APPROVED** [review]

**Comment by lrnt** [comment]
Leaving my ENS name for the ENS bounty: lrnts.eth

https://twitter.com/avsa/status/1468231064313339924

**Pull request merged**